src/sql/workbench/contrib/objectExplorer/browser/serverTreeView.ts

```diff
@@ -619,6 +622,13 @@ export class ServerTreeView extends Disposable implements IServerTreeView {
 					if (errorListener) {
 						errorListener.dispose();
 					}
+
+					this._telemetryService.createActionEvent(TelemetryKeys.TelemetryView.ObjectExplorer, TelemetryKeys.TelemetryAction.ObjectExplorerFilter)
+						.withAdditionalProperties({
+							filterPropertyNames: JSON.stringify(filters.map(f => f.displayName)),
```

Comment:
Is this only going to contain the filter properties the user set? And not include the empty ones?

Reply:
Yeah, it only includes the one which are set